src/main/java/uk/gov/hmcts/ccd/domain/model/common/DisplayContextParameterUtil.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.hmcts.ccd.domain.model.common;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import static java.util.stream.Collectors.joining;
+
+public class DisplayContextParameterUtil {
+    public enum Parameter {
+        COLLECTION("#COLLECTION"),
+        TABLE("#TABLE");
```

Comment:
don't think we need to define anything else other than COLLECTION. That's all Data Store should concern with

Reply:
done 👍